apps/mochi-web/components/base/dnd/DndWrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {
+  DragDropContext,
+  DragDropContextProps,
+  DroppableProps,
+} from 'react-beautiful-dnd'
+import { StrictModeDroppable } from './StrictModeDroppable'
+
+interface Props {
+  children: React.ReactNode
+  className?: string
+  componentProps: {
+    context: Omit<DragDropContextProps, 'children'>
+    droppable: Omit<DroppableProps, 'children'>
+  }
+}
+
+export const DndWrapper = ({ children, className, componentProps }: Props) => {
+  return (
+    <DragDropContext {...componentProps.context}>
```

Comment:
The end result of the order is incorrect, particularly regarding non-dragged items being reordered after a drop. Please check the video:

https://github.com/consolelabs/mochi-ui/assets/12707960/5f02f7f7-adf0-4ac1-8cf1-c57e0bdc9f1d